controller.xql

```diff
@@ -53,10 +54,10 @@ else if ($exist:resource eq "update.xql") then
 else if ($exist:path eq "/admin.html") then
     let $user := request:get-attribute("org.exist.public-repo.login.user")
     return
-        if ($user and (sm:is-dba($user) or "repo" = sm:get-user-groups($user))) then
+        if (exists($user) and sm:get-user-groups($user) = "repo") then
```

Comment:
I should probably update this to use the technique used elsewhere in the PR: Replace the hardcoded `"repo"` with `config:repo-descriptor()/repo:permissions/@group`.

Reply:
Fixed in https://github.com/eXist-db/public-repo/pull/53/commits/166cd0bea1b06d7da55146e8980217ed40106581.

Reply:
To make this information easier to access, I've added a function, `config:repo-premissions()`, which returns a map, so you can simply append `?user` or `?group` to access these values.